sources/router.c

```diff
@@ -348,14 +349,24 @@ od_router_status_t od_router_route(od_router_t *router, od_client_t *client)
 
 	/* match latest version of route rule */
 	od_rule_t *rule;
+
+	struct sockaddr_storage sa;
+	if (!client->is_watchdog) {
```

Comment:
Why do we exempt watchdog? Can we exampt all outgoing clients, not just watchdog?

Reply:
thanks, now we exempt all internal clients